### PR TITLE
Build and package a pyc file for every py file

### PR DESCRIPTION
**Pull request by smithfarm**
Fixes: https://github.com/SUSE/DeepSea/issues/531

### CODE REVIEW

**Comment by smithfarm** [comment]
This has been tested in the sense that I had OBS build it, and verified that the resulting RPM contains the `.pyc` files.

**Comment by theanalyst** [comment]
In the long run maybe we should make setup.py do these things since that is the normal way python packages are meant to be. This is fine for now

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
-*.py[co]
```
Comment (theanalyst) [inline .gitignore]: I still think we are not going to add these files to git, so should we keep the gitignore?

Comment (smithfarm) [inline .gitignore]: This line is a duplicate. Even after this change, `*.pyc` will continue to be ignored :-)

Comment (theanalyst) [inline .gitignore]: ah ok :)

**Pull request merged**